consensus/bor/finality/whitelist_helpers.go

```diff
@@ -33,7 +33,7 @@ func fetchWhitelistCheckpoint(ctx context.Context, heimdallClient heimdall.IHeim
 	// fetch the latest checkpoint from Heimdall
 	checkpoint, err := heimdallClient.FetchCheckpoint(ctx, -1)
 	if err != nil {
-		config.logger.Debug("Failed to fetch latest checkpoint for whitelisting", "err", err)
+		config.logger.Warn("Failed to fetch latest checkpoint for whitelisting", "err", err)
```

Comment:
The logs below are still `Debug`. Let's stay with a common pattern for all fetch functions.